Lib/multiprocessing/managers.py

```diff
@@ -8,8 +8,7 @@
 # Licensed to PSF under a Contributor Agreement.
 #
 
-__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token',
-            'SharedMemoryManager' ]
+__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token']
```

Comment:
`['BaseManager', 'SyncManager', 'BaseProxy', 'Token' ]`
or 
`[ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token' ]`
?

Reply:
Done in last commit.